README.md

```diff
@@ -16,16 +16,16 @@ Node.js - Javascript runtime environment ([nodejs.org](https://nodejs.org/en/))
 1. Clone this repository.
 1. Run ```npm install``` in the project root directory.
 1. Install PostgreSQL, start the PostgreSQL server, and connect to it via psql.
-1. In psql, run ```CREATE DATABASE environmental_dashboard;``` to create the database.
-1. Still in psql, run ```CREATE DATABASE environmental_dashboard_testing;``` to create a database for automated tests.
+1. In psql, run ```CREATE DATABASE oed;``` to create the database.
+1. Still in psql, run ```CREATE DATABASE oed_testing;``` to create a database for automated tests.
 1. Create a .env file in the root directory of the project with the following, replacing (?) with the desired information: <br>
-&nbsp;&nbsp;&nbsp;&nbsp;SERVER_PORT=? // The port that the server should run on. 3000 is a good default choice<br>
+&nbsp;&nbsp;&nbsp;&nbsp;SERVER_PORT=? // The port that the server should run on. `3000` is a good default choice<br>
```

Comment:
This really has nothing to do with the rest of the pull request, but I like this better as a code block:
```
SERVER_PORT=?              // The port that the server should run on. 3000 is a good default choice
DB_USER=?                  // The user that should be used to connect to postgres
DB_DATABASE=?              // The database you just created, so likely environmental_dashboard
DB_TEST_DATABASE=?         // The test database you just created, so likely environmental_dashboard_testing
DB_PASSWORD=?              // The password for your postgres user
DB_HOST=?                  // The host for your postgres db, likely localhost
DB_PORT=?                  // The port for your postgres db, likely 5432
TOKEN_SECRET=?             // Token for authentication. Generate something secure and random
```
Also I'm pretty sure the `<br>` tags are unnecessary.